protocols.md

```diff
@@ -0,0 +1,73 @@
+# Protocols
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
```

Comment:
See my comment on "protocol" as a term (feeling a little technologizing / scientizing), that said I'm totally down with gug of the idea of protocols

Reply:
Totally agree. What about stealing "guides" from enspiral?

Reply:
used "guides" though we're being a little more specific than enspiral treats guides